package.json

```diff
@@ -18,40 +18,46 @@
 	"main": "./dist/index.js",
 	"exports": "./dist/index.js",
 	"scripts": {
-		"prepare": "husky install",
+		"prepare": "husky",
 		"build": "rimraf dist && yarn build:main && yarn update:eslint-docs",
 		"build:main": "tsc -p tsconfig.build.json",
-		"lint:raw": "eslint --ext .ts --ext .js --ext .tsx --ext .jsx --ignore-pattern dist",
+		"lint:raw": "run eslint --ignore-pattern dist",
 		"lint": "yarn lint:raw .",
 		"lint-fix": "yarn lint --fix",
 		"test": "yarn lint && yarn unit",
 		"unit": "jest",
 		"license-validate": "yarn sofie-licensecheck --allowPackages path-scurry@1.6.1",
 		"update:eslint-docs": "eslint-doc-generator",
-		"validate:dependencies": "yarn audit --groups dependencies && yarn license-validate",
-		"validate:dev-dependencies": "yarn audit --groups devDependencies",
+		"validate:dependencies": "yarn npm audit --environment production && yarn license-validate",
+		"validate:dev-dependencies": "yarn npm audit --environment development",
 		"release": "sofie-version"
 	},
-	"dependencies": {},
 	"devDependencies": {
-		"@sofie-automation/code-standard-preset": "^2.3.1",
-		"@types/jest": "^29.4.1",
-		"@types/node": "^18.15.1",
-		"@typescript-eslint/utils": "^5.54.1",
-		"eslint-doc-generator": "^1.4.3",
-		"eslint-plugin-eslint-plugin": "^5.0.8",
-		"jest": "^29.5.0",
-		"rimraf": "^4.4.0",
-		"ts-jest": "^29.0.5",
-		"ts-node": "^10.9.1",
-		"typescript": "^4.9.5"
+		"@sofie-automation/code-standard-preset": "github:SuperFlyTV/sofie-code-standard-preset#commit=83e28ec8c6e0ea6287c20814f5dcb0cde1226b3d",
```

Comment:
Note to self: remember to update this to a proper version number after having merged the PR https://github.com/nrkno/sofie-code-standard-preset/pull/28